linux-tools/pexpect_test/pexpect_test.py

```diff
@@ -30,6 +30,10 @@ def run_once(self, test_path=''):
         """
         try:
             os.environ["LTPBIN"] = "%s/shared" %(test_path)
+            cwd = os.getcwd()
+            os.chdir("%s/pexpect_test" %(test_path))
```

Comment:
please take this opportunity to use `os.path.join` here and elsewhere,
`pexpect_test_path = os.path.join(test_path, "pexpect_test")`
`os.chdir(pexpect_test_path)`

`ret_val = subprocess.call(os.path.join(pexpect_test_path, 'pexpect.sh'), shell=True)`